force-app/main/default/classes/HOT_ContentDocumentLinkService.cls

```diff
@@ -0,0 +1,16 @@
+public with sharing class HOT_ContentDocumentLinkService {
+    public static void setContentDocumentLinks(Map<Id, List<ContentDocumentLink>> serviceAppointmentMap) {
```

Comment:
argumentet kan gjerne hete contentDocumentLinksByServiceAppointment.
Kan godt kalle Map'sa du lager i triggerne for det samme. Mer informativt.

Reply:
Utført